DotNetRu.Android/Resources/menu/nav_menu.xml

```diff
@@ -1,33 +1,34 @@
-﻿<?xml version="1.0" encoding="UTF-8" ?>
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file needed?

Reply:
No, it's part of bottom menu. I think we should delete it and all related things in separate pr (with other not needed things, like pages for uwp)

Reply:
Ok